src/components/PurchaseOrder/PODetails/PODetailsForm.js

```diff
@@ -271,7 +271,7 @@ class PODetailsForm extends Component {
               component={Checkbox}
               fullWidth
               label={<FormattedMessage id="ui-orders.orderDetails.manualPO" />}
-              name="manual_po"
+              name="manualPo"
```

Comment:
did you miss assigned_to_user above (line 257)?

Reply:
no, it's by intention, ui-only fields.